staging/src/k8s.io/apiserver/pkg/util/flowcontrol/default-objects.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	rmv1a1 "k8s.io/api/flowcontrol/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// DefaultPriorityLevelConfigurationObjects returns the instances of
+// the ones defined in the KEP, with system-top in position 0 and
+// workload-low in position 1
+func DefaultPriorityLevelConfigurationObjects() []*rmv1a1.PriorityLevelConfiguration {
```

Comment:
we can split `ImmutabledDefaults` from `DefaultPriorityLevelConfigurationObjects`, and only assumes the immutable set on launching the server. the immutables cannot be deleted as forbidden from model's validation. i can imagine the `ImmutabledDefaults` consists of a `system-top-loopback` flow-schema and a `system-top` exempt priority and a `workload-low` catch-all priority

Reply:
I expect that loopback connections are a proper _subset_ of those that will be matched by our default `system-top` FlowSchema.  I am uncomfortable exempting only a subset during startup.

I considered having the startup behavior use just two priority levels and two flow schemas, one for exempt and one for non-exempt.  But then I thought, why use such impoverished behavior when we have a hypothesis of what a good config looks like?  That is why I proposed using our hypothesis during startup.